src/main/java/com/mraof/minestuck/skaianet/UnderlingController.java

```diff
@@ -59,20 +63,20 @@ public static GristType getUnderlingType(UnderlingEntity entity)
 		else
 		{
 			impWeight = 8 - (difficulty - 8)/3;
-			if(difficulty < 20)
```

Comment:
The values related to difficulty here are in a sense hardwired, so it's insufficient to just change numbers in the if-statements (it actually breaks up the difficulty-to-weight functions such that they're no longer continuous functions). You'd have to also adapt the actual difficulty-to-weight functions, and to handle the difficulty cap.
However if you want to tweak these numbers in closer detail like you've tried to do, I'd rather suggest you change this weight code entirely into something that is better designed to be tweaked with.